From 46cb37b4b2efdcc8fb9d2edf8e08a3b0e1c15f80 Mon Sep 17 00:00:00 2001
From: Joel A Fernandes <agnel.joel@gmail.com>
Date: Tue, 21 Jun 2011 15:53:32 -0500
Subject: [PATCH 3/4] DMAI: Disable display rotation to avoid problems due to not passing vrfb in bootargs

Disable display rotation as it is not known to be used by many users and can result
in problems if vrfb is not passed in bootargs.

Signed-off-by: Joel A Fernandes <agnel.joel@gmail.com>
---
 linux/omap3530/Display_v4l2.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/packages/ti/sdo/dmai/linux/omap3530/Display_v4l2.c b/packages/ti/sdo/dmai/linux/omap3530/Display_v4l2.c
index 61f15bb..3cfd75d 100644
--- a/packages/ti/sdo/dmai/linux/omap3530/Display_v4l2.c
+++ b/packages/ti/sdo/dmai/linux/omap3530/Display_v4l2.c
@@ -174,6 +174,8 @@ Display_Handle Display_v4l2_create(BufTab_Handle hBufTab, Display_Attrs *attrs)
         return NULL;
     }
     
+#undef V4L2_CID_ROTATE
+
 #if defined(V4L2_CID_ROTATE)
     /* Make rotation by default 0 for DVI and S-video*/
 
-- 
1.7.0.4

